Lib/test/test_struct.py

```diff
@@ -599,6 +599,17 @@ def test_boundary_error_message_with_negative_offset(self):
                 'offset -11 out of range for 10-byte buffer'):
             struct.pack_into('<B', byte_list, -11, 123)
 
+    def test_boundary_error_message_with_large_offset(self):
+        # Test overflows cause by large offset and value size (issue 30245)
+        regex = (
+            "pack_into requires a buffer of at least " + str(sys.maxsize + 4) +
+            " bytes for packing 4 bytes at offset " + str(sys.maxsize) +
+            " \(actual buffer size is 10\)"
```

Comment:
Use raw string literals. Compiling `"\("` emits a deprecation warning.

You also could use classic string formatting, string's format() method, or f-string expression for composing this string.